### PR TITLE
Handling loop breaks and returns

### PR DESCRIPTION
**Pull request by pratikbhd**
This PR implements the following changes:
1.  Handling restructuring of breaks and returns inside loops

Note: Please do not delete this branch after merging since any updates on this will be done through this branch.

### CODE REVIEW

File: delphi/translators/for2py/loop_handle.py [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+
+class RefactorBreaks(object):
+    """This class defines the refactor state of the intermediate AST while
+    removing the breaks and returns inside while loops
+    """
+
+    def __init__(self):
+        self.find_opp = {
+            ".eq.": ".ne.",
```
Comment (skdebray) [inline delphi/translators/for2py/loop_handle.py]: This table of complementary operators also appears in rectify.py.  Can we pull this out into syntax.py?

File: delphi/translators/for2py/loop_handle.py [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+        self.is_break = False
+        self.shifted_items = {}
+        self.shifted_level = 0
+
+    def refactor(self, ast: Dict) -> Dict:
+        body = ast["ast"][0]["body"]
+        self.search_while(body)
+
+        # self.search_breaks(body)
+        ast["ast"][0]["body"] = body
```
Comment (skdebray) [inline delphi/translators/for2py/loop_handle.py]: I don't understand lines 47 and 51.

File: delphi/translators/for2py/loop_handle.py (hunk 1 above)
Comment (pratikbhd) [reply delphi/translators/for2py/loop_handle.py]: Just put the dictionary into `syntax.py` and updated their usage here and in `rectify.py`

File: delphi/translators/for2py/loop_handle.py (hunk 2 above)
Comment (pratikbhd) [reply delphi/translators/for2py/loop_handle.py]: This is redundant and not required. Removed line 51.

**Pull request merged**